fanficfare/adapters/adapter_deviantartcom.py

```diff
@@ -159,7 +160,12 @@ def extractChapterUrlsAndMetadata(self):
         # self.story.setMetadata('status', 'Completed')
 
         pubdate = soup.select_one('time').get_text()
-        self.story.setMetadata('datePublished', makeDate(pubdate, '%b %d, %Y'))
+
+        # Maybe do this better, but this works
+        try:
+            self.story.setMetadata('datePublished', makeDate(pubdate, '%b %d, %Y'))
+        except:
+            self.story.setMetadata('datePublished', makeDate(str(parse_relative_date_string(pubdate)), '%b %d, %Y'))
```

Comment:
`parse_relative_date_string()` returns a `datetime` object, no need to call `makeDate()`.

(And FFF code shouldn't use `str()` because we're still supporting Python 2 in older Calibre versions.)